doc/decisions/frontend_state_management.md

```diff
@@ -0,0 +1,31 @@
+# Frontend State Management Library
+
+## Problem
+
+PermaplanT is a web application built with React that requires efficient and flexible state management.
+
+## Constraints
+
+- The state management library must support React and work seamlessly with the rest of the application's technology stack.
+- The library should be well-maintained, have a large community of developers, and a good documentation.
+
+## Considered Alternatives
+
+- [Redux](https://redux.js.org/)
```

Comment:
Again, please a sentence for each alternative why it is not taken.
Which features do they have and why don't we need them?